lib/chef/knife/bootstrap.rb

```diff
@@ -384,18 +385,28 @@ class Bootstrap < Knife
       }.freeze
 
       DEPRECATED_FLAGS.each do |deprecated_key, deprecation_entry|
-        new_key, deprecated_long, replacement_value = deprecation_entry
+        new_key, deprecated_long, replacement_value, deprecated_short = deprecation_entry
```

Comment:
I think this is good for now, but it's getting unwieldy. I think we're at the point where if we add more, it might make sense to have a DeprecatedOption struct. 


With  I'm POCing some mixlib-install work that will manage deprecation behaviors there; but this is good for now.